### ISSUE TITLE
peerDependencies warning

### ISSUE THREAD

**Issue by JoshSmith** [sir-dunxalot/ember-tooltips#73]
```
npm WARN peerDependencies The peer dependency ember-tether@0.3.1 included from ember-tooltips will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
```

Does this mean we should start depending on the dependency explicitly ourselves?

**Comment by nbibler** [comment]
I came here to add an issue around this, as well. It looks as though `ember-tether` is defined as a peer and development dependency, but not an actual production-time dependency. Both Components provided by this library, however, require that `ember-ether` be installed in order to load (via require and extend calls).

Without explicitly adding `ember-tether` to my application's `package.json` file, this addon breaks my app.

**Comment by sir-dunxalot** [comment]
Ok, it looks like #60 wasn't fixed as expected and we should be able to fix this by adding `ember-tether` to `dependencies` as well as `peerDependencies`. This shouldn't be top hard to fix and I will do it before the upcoming `2.0.0` release.

**Comment by sir-dunxalot** [comment]
Of course, fi anyone wants to submit a PR in the meantime I'll be happy to merge and release.

**Comment by keradus** [comment]
Any news here perhaps ?

**Comment by JoshSmith** [comment]
Also wondering what's going on here. Looks like `2.0` went without any update to `dependencies`.

**Comment by sir-dunxalot** [comment]
Checking upstream in ember-tether to see what they can do. Otherwise, I wouldn't expect this warning to disappear anytime soon. Sorry, guys.

**Comment by sir-dunxalot** [comment]
With the latest version of `ember-tether` (`0.4.0`) I think we can remove ember-tether from the `peerDependencies` list entirely. I need to confirm this but that would solve this issue.

**Comment by a15n** [comment]
I think that `ember-tether: '0.3.1'` may be needed for ember versions before 2.4. I haven't looked too far into it yet but wanted to raise that here 

https://github.com/yapplabs/ember-tether

**Comment by sir-dunxalot** [comment]
Fixed by [2.9.0-beta.0](https://github.com/sir-dunxalot/ember-tooltips/releases/tag/2.9.0-beta.0).

**Comment by JoshSmith** [comment]
🎉  thanks @sir-dunxalot!
